backend/src/main/java/pl/edu/agh/awiteks_backend/models/User.java

```diff
@@ -127,4 +155,9 @@ public List<ForumThread> getForumThreadList() {
     public void setForumThreadList(List<ForumThread> forumThreadList) {
         this.forumThreadList = forumThreadList;
     }
+
+    public boolean isFollowing(ForumThread thread) {
+        //return followedThreads.contains(thread);
+        return false;
+    }
```

Comment:
Pasuje usunąć ten komentarz i dodać komentarz z `TODO`, dlaczego jest to zharddkorowane